<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ethernet to SubG Demo</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>resources/ico/logo_1_8X3_icon.ico</normaloff>resources/ico/logo_1_8X3_icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="2,0">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
         </font>
        </property>
        <property name="title">
         <string>Devices</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTableView" name="tableViewDevices">
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="verticalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
         </font>
        </property>
        <property name="title">
         <string>Sensors/Controls</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
            </font>
           </property>
           <property name="text">
            <string>Illuminance</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QProgressBar" name="progressBarIlluminance">
           <property name="value">
            <number>24</number>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
            </font>
           </property>
           <property name="text">
            <string>Humidity</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
            </font>
           </property>
           <property name="text">
            <string>Temperature</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
            </font>
           </property>
           <property name="text">
            <string>LED Switch</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="pushButtonLedToggle">
             <property name="font">
              <font>
               <family>Microsoft YaHei UI</family>
              </font>
             </property>
             <property name="text">
              <string>Turn On</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="3" column="1">
          <widget class="QProgressBar" name="progressBar_2Humidity">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QProgressBar" name="progressBar_3Temperature">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
        </layout>
        <zorder>label</zorder>
        <zorder>progressBarIlluminance</zorder>
        <zorder>label_2</zorder>
        <zorder>progressBar_2Humidity</zorder>
        <zorder>label_3</zorder>
        <zorder>progressBar_3Temperature</zorder>
        <zorder>verticalSpacer</zorder>
        <zorder>verticalSpacer_2</zorder>
        <zorder>verticalSpacer_3</zorder>
        <zorder>label_4</zorder>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
         </font>
        </property>
        <property name="title">
         <string>Communication</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit">
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="plainText">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="lineEditData">
             <property name="font">
              <font>
               <family>Microsoft YaHei UI</family>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonSend">
             <property name="font">
              <font>
               <family>Microsoft YaHei UI</family>
              </font>
             </property>
             <property name="text">
              <string>Send</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>836</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_2"/>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_2">
   <property name="text">
    <string>About</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>toggleSend(bool)</slot>
  <slot>validateData(QString)</slot>
  <slot>lockInputItems(bool)</slot>
  <slot>validateData()</slot>
  <slot>refreshUartPorts()</slot>
  <slot>lockUartPort(bool)</slot>
  <slot>connect(bool)</slot>
  <slot>readRfData()</slot>
  <slot>writeRfData()</slot>
  <slot>clearPacketList()</slot>
  <slot>clearTxStatistics()</slot>
  <slot>clearRxStatistics()</slot>
 </slots>
</ui>
